.yarnrc.yml

```diff
@@ -1 +1,3 @@
 nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
```

Comment:
`.yarn` is part of the list inside `.gitignore`, so the files inside of `.yarn` are not part of the commited repo. 

While trying to run `yarn install` or any yarn command I get this error: 
```
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '/Users/andresrodriguez/Visual_Studio_Code/kilt-distillery-cli/.yarn/releases/yarn-3.6.4.cjs'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

This is very inconvenient.

Reply:
This is a problem with your Yarn version